lib/jxl/dec_16bit_reconstruct.cc

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) the JPEG XL Project Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+#include "lib/jxl/dec_16bit_reconstruct.h"
+
+#undef HWY_TARGET_INCLUDE
+#define HWY_TARGET_INCLUDE "lib/jxl/dec_16bit_reconstruct.cc"
+#include <hwy/foreach_target.h>
+#include <hwy/highway.h>
+
+#include "lib/jxl/dec_xyb-inl.h"
+
+HWY_BEFORE_NAMESPACE();
+namespace jxl {
+namespace HWY_NAMESPACE {
+void FinalizeImageRect16(Image3F* input_image, const Rect& input_rect,
```

Comment:
Any chance we can test that applying gaborish with this is ~the same as applying it with the filters functions in a unittest only enabled when NEON is compiled/supported?